androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/SeekClickableArea.kt

```diff
@@ -68,7 +68,7 @@ fun SeekClickableArea(
         if (isTapCountGreaterThanZero) {
             val timeLabel = getSeekTime()
             Column(horizontalAlignment = Alignment.CenterHorizontally) {
-                seekIcon(modifier = Modifier.scale(scaleAnimation))
+                seekIcon(modifier = Modifier.scale(scaleAnimation()))
```

Comment:
As discussed [here](https://developer.android.com/jetpack/compose/performance/bestpractices#defer-reads), prefer using the lambda versions of the modifiers. In this case I think that can be `graphicsLayer`